app/src/main/java/com/everest/moviedb/RetrofitClient.kt

```diff
@@ -0,0 +1,20 @@
+package com.everest.moviedb
+
+import com.everest.moviedb.model.MovieService
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitClient {
+    var movieService: MovieService? = null
+
+    const val BASE_URL = R.string.API_BaseUrl.toString()
+
+    fun getClient(): MovieService {
```

Comment:
small thing. It creates new client everytime we call getClient. we can create single instance and use it